core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-tables/contributor-admin-stats-table.component.spec.ts

```diff
@@ -844,5 +893,57 @@ describe('Contributor stats component', () => {
       activeTab = component.TAB_NAME_QUESTION_REVIEWER;
       expect(component.getContributionCount(activeTab, 1, 2, 3, 4)).toEqual(4);
     }));
+
+    it('should convert the items per page dropdown back into numbers', fakeAsync(() => {
+      component.onItemsPerPageChange('05');
+      expect(component.itemsPerPage).toEqual(5);
+    }));
+  });
+
+  describe('when more than one page of stats are returned', () => {
+    var createContributorStats = function createContributorStats(
+      numStats: number
+    ): QuestionReviewerStats[] {
+      let stats: QuestionReviewerStats[] = [];
+      for (let i = 0; i < numStats; i++) {
+        stats.push(questionReviewerStats);
+      }
+      return stats;
+    };
```

Comment:
Remove the `var createContributorStats = ` part. Also, there are other tests you've added in this PR that populate question reviewer stats. They should call this new helper function as well.

Reply:
I can't do it as a function declaration as there is a linting error "Expecting a function expression"   

![image](https://github.com/oppia/oppia/assets/123914818/75006f44-e94f-4b40-82d1-376dda490eff)

I have changed the other set up calls to use the function though.

Reply:
@stevenjgodfrey Ah, my bad. Use a function expression, but don't repeat the function name twice. So instead of
```ts
const createContributorStats = function createContributorStats(numberOfStats: number): QuestionReviewerStats[] {...};
```
write it like this:
```ts
const createContributorStats = function (numberOfStats: number): QuestionReviewerStats[] {...};
```